### DISCUSSION TITLE
Support for Eltako window contact FTKE with EEP F6-10-00 [DONE]

### DISCUSSION THREAD

**Discussion by stefanhofmann2** [mak-gitdev/HA_enoceanmqtt#29] [Request] [answered]
Can you please add the following sensor:
Eltako window contact FTKE
Radio telegrams
according to EEP F6-10-00
ORG = 0x05
Data_byte3 = 0xF0 = window closed
0xE0 = window open

I have just used the mapping file in the develop branch and it works fine.
Thanks

**Comment by mak-gitdev** [comment, marked as answer]
Hi @stefanhofmann2,

Support is already available. See this [thread](https://github.com/mak-gitdev/HA_enoceanmqtt/discussions/6).

You just have to define your sensor in your device configuration file.
Examples are available in this [example file](https://github.com/mak-gitdev/HA_enoceanmqtt-addon/blob/master/addon-dev/enoceanmqtt.devices.sample) if you are using the addon.

It will be something like:
```
[my_FTKE]
address = <address of your FTKE>
rorg = 0xF6
func = 0x10
type = 0x00
```
Replace `my_FTKE` with whatever string you want (do not use space character). It will be use to create your device name.

After that, restart HA_enoceanmqtt and you will see your device in Home Assistant.

Do not hesitate if you have questions regarding above information.
